diff -Nur old/src/main.conf new/src/main.conf
--- old/src/main.conf	2010-08-25 13:10:02.000000000 +0800
+++ new/src/main.conf	2010-09-26 17:28:39.000000000 +0800
@@ -32,10 +32,10 @@
 
 # What value should be assumed for the adapter Powered property when
 # SetProperty(Powered, ...) hasn't been called yet. Defaults to true
-InitiallyPowered = true
+InitiallyPowered = false
 
 # Remember the previously stored Powered state when initializing adapters
-RememberPowered = true
+RememberPowered = false
 
 # Use vendor, product and version information for DID profile support.
 # The values are separated by ":" and VID, PID and version.
